scalafmt-tests/src/test/scala/org/scalafmt/FormatTests.scala

```diff
@@ -54,6 +54,9 @@ class FormatTests extends FunSuite with CanRunTests with FormatAssertions {
         e.partialOutput
       case x => x.get
     }
+    if (!onlyManual) {
+      assertEquals(obtained, t.expected, "Obtained formatting output does not match the expected formatting output" )
```

Comment:
`assertEquals` produces copy-pasteable output instead of `.stripMargin` formatted multiline strings from `assertNoDiff`